content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -75,7 +75,9 @@ This command does two things:
 
 <SwitchTech technologies={['*', 'planetscale']}>
 
-You are now ready to push your new schema to the `develop` branch of your database. To do this, use the `db push` CLI command:
+You are now ready to push your new schema to your database. PlanetScale does not allow direct pushing of schema changes to the `main` database branch, so you will need to use a different branch, or create a new one. As an example, create a `develop` branch from `main` using the 'New branch' option on the overview page for your database, and connect to it using the instructions in [Connect your database](/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-planetscale/).
```

Comment:
Maybe without the trailing slash? The page you want to link to seems to be https://deploy-preview-2625--prisma2-docs.netlify.app/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-planetscale on the existing preview deployment @keerlu

Reply:
Still not working, not sure why

Reply:
No idea then, might be a bug (most probably in context of the tech switcher pages). Not sure if we have working internal links in here anywhere.

Reply:
No, I couldn't see any. What's the best idea for a workaround - using the full link maybe, or rephrasing somehow and not having a link at all?

Reply:
A relative link could also work: `../connect-your-database-typescript-planetscale` maybe.

Reply:
Still doesn't work for some reason

Reply:
Then I would just put an absolute external link in there until we can figure that out.

Reply:
Done